overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-check-luks-syntax/rhcos-fail-boot-for-older-luks-syntax.service

```diff
@@ -0,0 +1,15 @@
+# Fixes the encryption issue with LUKS in 4.7 when a user 
+# tries to use the RHCOS 4.6 or earlier LUKS syntax in an 
+# Ignition config.
```

Comment:
"Fixes the encryption issue" doesn't really explain anything.  In fact, what we're doing is failing the boot if the user tries to do that, so they don't accidentally end up with an unencrypted system.